distributed/bokeh/scheduler_html.py

```diff
@@ -5,6 +5,7 @@
 from tornado import escape
 from tornado import gen
 from tornado import web
+from prometheus_client import Gauge, generate_latest
```

Comment:
We'll want to put this import within the prometheus handler I think so that the library doesn't strictly depend on it (we try to keep dependencies lightweight).